proto/v1/activity_service.proto

```diff
@@ -0,0 +1,213 @@
+    ACTION_ISSUE_COMMENT_CREATE = 6;
+    // ACTION_ISSUE_FIELD_UPDATE is the type for updating an issue's field.
+    ACTION_ISSUE_FIELD_UPDATE = 7;
+    // ACTION_ISSUE_STATUS_UPDATE is the type for updating an issue's status.
+    ACTION_ISSUE_STATUS_UPDATE = 8;
+
+    // ACTION_PIPELINE_STAGE_STATUS_UPDATE is the type for stage begins or ends.
+    ACTION_PIPELINE_STAGE_STATUS_UPDATE = 9;
+    // ACTION_PIPELINE_TASK_STATUS_UPDATE is the type for updating pipeline task status.
+    ACTION_PIPELINE_TASK_STATUS_UPDATE = 10;
+    // ACTION_PIPELINE_TASK_FILE_COMMIT is the type for committing pipeline task files.
+    ACTION_PIPELINE_TASK_FILE_COMMIT = 11;
+    // ACTION_PIPELINE_TASK_STATEMENT_UPDATE is the type for updating pipeline task SQL statement.
+    ACTION_PIPELINE_TASK_STATEMENT_UPDATE = 12;
+    // ACTION_PIPELINE_TASK_EARLIEST_ALLOWED_DATE_UPDATE is the type for updating pipeline task the earliest allowed time.
+    ACITON_PIPELINE_TASK_EARLIEST_ALLOWED_DATE_UPDATE = 13;
+
+    // ACTION_PROJECT_MEMBER_CREATE is the type for creating a new project member.
+    ACTION_PROJECT_MEMBER_CREATE = 14;
+    // ACTION_PROJECT_MEMBER_ROLE_UPDATE is the type for updating a project member's role.
+    ACTION_PROJECT_MEMBER_ROLE_UPDATE = 15;
+    // ACTION_PROJECT_MEMBER_DELETE is the type for deleting a project member.
+    ACTION_PROJECT_MEMBER_DELETE = 16;
+
+    // ACTION_PROJECT_REPOSITORY_PUSH is the type for pushing to a project repository.
+    ACTION_PROJECT_REPOSITORY_PUSH = 17;
+
+    // ACTION_PROJECT_DATABASE_TRANSFER is the type for transferring a database to a project.
+    ACTION_PROJECT_DTABASE_TRANSFER = 18;
+    
+    // ACTION_PROJECT_DATABASE_RECOVERY_PITR_DONE is the type for database PITR recovery done.
+    ACTION_PROJECT_DATABASE_RECOVERY_PITR_DONE = 19;
+
+    // Both in workspace and project resource.
+    //
+    // ACTION_SQL_EDITOR_QUERY is the type for SQL editor query.
+    // If user runs SQL in Read-only mode, this action will belong to project resource.
+    // If user runs SQL in Read-write mode, this action will belong to workspace resource.
+    ACTION_SQL_EDITOR_QUERY = 20;
+  }
+  Action action = 5 [(google.api.field_behavior) = REQUIRED];
+
+  enum Level {
+    LEVEL_UNSPECIFIED = 0;
+    // LEVEL_INFO is the type for information.
+    LEVEL_INFO = 1;
+    // LEVEL_WARNING is the type for warning.
+    LEVEL_WARNING = 2;
+    // LEVEL_ERROR is the type for error.
+    LEVEL_ERROR = 3;
+  }
+  Level level = 6 [(google.api.field_behavior) = OUTPUT_ONLY];
+
+  // The container of the activity.
+  // Format:
+  // For ACTION_MEMBER_*: user:emailid
+  // For ACTION_ISSUE_*: issues/{issue}
+  // For ACTION_PIPELINE_*: pipelines/{pipeline}
+  // For ACTION_PROJECT_*: projects/{project}
+  // For ACTION_SQL_EDITOR_QUERY: workspaces/{workspace} OR projects/{project}
+  string container = 7 [(google.api.field_behavior) = REQUIRED];
+
+  string comment = 8;
```

Comment:
What's this comment used for?

Reply:
We have comment for some activities, e.g. https://sourcegraph.com/github.com/bytebase/bytebase/-/blob/backend/runner/taskrun/executor.go?L412.

But frontend seems doesn't use it. Let's remove it.